src/utils/utils.tsx

```diff
@@ -0,0 +1,5 @@
+export const converToString = (time: number): string => {
```

Comment:
`convertToString`....convert what to string? what kind of string?

if you're going to put anything in a util, it should be as self-documenting as possible (or even add something like https://jsdoc.app/